go/runtime/history/db.go

```diff
@@ -29,6 +30,14 @@ var (
 	//
 	// Value is CBOR-serialized roothash.RoundResults.
 	roundResultsKeyFmt = keyformat.New(0x03, uint64(0))
+	// stakingEventsKeyFmt is the round staking events index key format.
+	//
+	// Value is CBOR-serialized list of staking.Events.
+	stakingEventsKeyFmt = keyformat.New(0x04, uint64(0))
+	// pendingStakingEventsKeyFmt is the per height pending staking events index key format.
+	//
+	// Value is a CBOR-serialized list of staking.Events.
+	pendingStakingEventsKeyFmt = keyformat.New(0x05, int64(0))
```

Comment:
Similar as above, should probably be generalized for arbitrary indexed events (probably different kinds of events don't need to be stored under different keys).